tools/stats/print_test_stats.py

```diff
@@ -698,7 +698,7 @@ def build_info() -> ReportMetaMeta:
         "build_sha1": os.environ.get("CIRCLE_SHA1", ""),
         "build_base_commit": get_base_commit(os.environ.get("CIRCLE_SHA1", "HEAD")),
         "build_branch": os.environ.get("CIRCLE_BRANCH", ""),
-        "build_job": os.environ.get("JOB_BASE_NAME", ""),
+        "build_job": os.environ.get("JOB_BASE_NAME", os.environ.get("CIRCLE_JOB", "")),
```

Comment:
Before we had some random None folders in our S3 when JOB_BASE_NAME wasn't set for CircleCI jobs--this handles the meta info for scribe for BC so that we do not accidentally introduce nulls to our tables when we enable reporting for more jobs potentially.